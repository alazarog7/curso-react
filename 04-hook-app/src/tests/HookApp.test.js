import React from 'react';
import shallow from "enzyme/build/shallow"
import { HooksApp } from '../HooksApp';
describe('Simple Test', () => {
    test('should return a snapshot', () => {
       const wrapper = shallow(<HooksApp/>);
       expect(wrapper).toMatchSnapshot();
    });
    
})
