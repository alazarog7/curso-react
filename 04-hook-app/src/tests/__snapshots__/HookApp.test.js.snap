// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple Test should return a snapshot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
